ex03.ex

```diff
@@ -59,8 +59,17 @@ defmodule Ex03 do
 
   """
 
-  def odd_even . . . "your code"
+  def odd_even( [] ), do: []
 
+  def odd_even( list ) do
```

Comment:
# -1 (not idiomatic)

Rather than splitting the list on line 65, most Elixir programmers would do it in the function definition.

# -1 for too much logic

Can you think of a way of doing this without an `if`?